lib/foodsoft_config.rb

```diff
@@ -260,6 +265,7 @@ def get_default_config
         tasks_period_days: 7,
         tasks_upfront_days: 49,
         shared_supplier_article_sync_limit: 200,
+        api_key: '',
```

Comment:
This can be empty, right? The default value of `nil` also works. This section is primarily for being backward-compatible, for items that really can't be `nil`. I'd rather put sane defaults in the app_config sample.